crates/kitsune2/tests/blocks.rs

```diff
@@ -339,17 +446,31 @@ async fn message_block_count_increases_correctly() {
         .await
         .unwrap();
 
-    // Check that the blocks count goes up on Carols side
```

Comment:
Replacing the prior logic here that did not distinguish between incoming and outgoing messages in the blocks count.